### PR TITLE
Made create-news UI

### PR DESCRIPTION
**Pull request by Gusvarov**
Made UI for #505

### CODE REVIEW

File: src/app/component/general/eco-news/create-news/create-news.component.css [hunk 1]
```diff
@@ -1,31 +1,150 @@
-a {
-    text-decoration: none;
```
Comment (nabermi2020) [inline src/app/component/general/eco-news/create-news/create-news.component.css]: General: all fonts should be implemented according to mockup

File: src/app/component/general/eco-news/create-news/create-news.component.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { Component, OnInit } from '@angular/core';
 })
 export class CreateNewsComponent implements OnInit {
 
+  link: string = 'https://cdn3.iconfinder.com/data/icons/popular-services-brands/512/angular-js-256.png';
```
Comment (nabermi2020) [inline src/app/component/general/eco-news/create-news/create-news.component.ts]: Please move it to links.ts as a constant

File: src/app/component/general/eco-news/create-news/create-news.component.css (hunk 1 above)
Comment (Gusvarov) [reply src/app/component/general/eco-news/create-news/create-news.component.css]: fixed

File: src/app/component/general/eco-news/create-news/create-news.component.ts (hunk 2 above)
Comment (Gusvarov) [reply src/app/component/general/eco-news/create-news/create-news.component.ts]: fixed

**Pull request merged**